src/main/java/net/sf/jabref/Groups.uml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Diagram>
+  <ID>JAVA</ID>
+  <OriginalElement>net.sf.jabref.model.groups.AbstractGroup</OriginalElement>
```

Comment:
Why not using [PlantUML](https://plugins.jetbrains.com/plugin/7017) for documenting?

Which tool did you use?